stringer.go

```diff
@@ -0,0 +1,30 @@
+package dig
+
+import (
+	"bytes"
+	"fmt"
+)
+
+// String representation of the entire Container
+func (c *Container) String() string {
```

Comment:
Why these functions are in a separate file? The different receiver type((pointer and value) in a String methods makes me to look in the other methods implementations.